verif/core_rrv/tb/core_rrv_verif_list.f

```diff
@@ -1,4 +1,4 @@
 +incdir+../../../source/common/
 ../../../verif/rv32i_ref/tb/rv32i_ref_pkg.sv
 ../../../verif/rv32i_ref/tb/rv32i_ref.sv
-../../../verif/core_rrv/tb/core_rrv_tb.sv
+../../../verif/core_rrv/tb/core_rrv_no_ref_tb.sv
```

Comment:
I think this is really bad.
having the "core_rrv_no_ref_tb.sv" **Instead**  of "core_rrv_tb.sv" for all cases is defenetly wrong..
we should have both:
../../../verif/core_rrv/tb/core_rrv_no_ref_tb.sv
../../../verif/core_rrv/tb/core_rrv_tb.sv

And in the builder when running tests that the reference model cannot check the CPU, we need to add "-top core_rrv_no_ref_tb"
But the defualt case should be using "core_rrv_tb.sv"